CHANGELOG.md

```diff
@@ -23,6 +24,8 @@ and [Keep a Changelog](http://keepachangelog.com/).
 ### Fixed
 
 - Fixed example not using correct array key for invalid login errors ([#148])
+- Prevented warning in `Wikimate::logout()` about `token` parameter that was unsupported
+  before MediaWiki v1.34 ([#151])
```

Comment:
I'd have preferred this to be in a separate commit (ideally it would have been part of the previous PR, but that ship has sailed). However, it's such a minor thing that I don't think it's worth the hassle to address now.